UnitTests.NUnit2/MediumTrustSqlBackendIntegrationTest.cs

```diff
@@ -21,6 +21,7 @@
 using NUnit.Framework;
 using Remotion.Linq.Development.UnitTesting.Sandboxing;
 
+// ReSharper disable once CheckNamespace
```

Comment:
I was unsure about this myself, I also prefer to leave it in the root, but the thing is that the test searches for assemblies with the same namespace as the current type, if I change the namespace, no tests are found - a simple solution would be to change `where t.Namespace == typeof (MediumTrustSqlBackendIntegrationTest).Namespace` to `typeof(SomeOtherClassInTheSameNamespace)`. What do you think?

Reply:
Yes changing the type to something else is better than moving the class back into the folder again. Maybe we can fix the bug at the same time since we are touching the line the causes it already :p